.github/workflows/pnp-input-api-build.yml

```diff
@@ -67,7 +67,17 @@ on:
         type: boolean
         required: false
         default: false
-        
+      service-name:
```

Comment:
We have to move this parameter to secrets. 
It should not be required.
Let's use specific names like `wid-token-audience` and change the description to `The audience of the generated Workload Identity token. Should be the name of the receiver service.`.